Disable ephemeral storage in Katib Config (#2315)
kubeflow/website: content/en/docs/components/katib/katib-config.md

   - `cpu = 500m`
   - `ephemeral-storage = 5Gi`

   You can run your metrics collector container without requesting
   the `ephemeral-storage` resource from the Kubernetes cluster.

>    For instance, if you want to use
>    [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
>    for your Katib experiments. To remove the `ephemeral-storage` resource from
>    the metrics collector container, set the negative values for the
>    `ephemeral-storage` requests and limits in your Katib config:

Reviewer: This sentence looks like it starts with "if" and doesn't end.

Maybe you can try this, if I understood the logic correctly:

```suggestion
   For instance, when using the
   [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
   for your Katib experiments, you can remove the `ephemeral-storage` resource from
   the metrics collector container by setting the negative values for the
   `ephemeral-storage` requests and limits in your Katib config as follows:
```

WDYT?

   - `cpu = 500m`
   - `ephemeral-storage = 5Gi`

   It is possible to not request the `ephemeral-storage` resource for the
   metrics collector's container. For example to use
   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
   To do that, set the negative values for the `ephemeral-storage` requests and

>    limits in your Katib config:

Reviewer: The second sentence is not complete, I think.
The third sentence is not clear about what "that" is.
Can you revise them please?

Author: Sure, @8bitmp3.
What do you think about this:
```
 You can run your metrics collector container without requesting
 the `ephemeral-storage` resource from the Kubernetes cluster.
 For instance, if you want to use
 [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
 for your Katib experiments. To remove the `ephemeral-storage` resource from
 the metrics collector container, set the negative values for the
 `ephemeral-storage` requests and limits in your Katib config:
```

   - `cpu = 500m`
   - `ephemeral-storage = 5Gi`

   It is possible to not request the `ephemeral-storage` resource for the
   suggestion's container. For example to use
   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
   To do that, set the negative values for the `ephemeral-storage` requests and

>    limits in your Katib config:

Reviewer: Same as https://github.com/kubeflow/website/pull/2315/files#r521391860

   - `cpu = 500m`
   - `ephemeral-storage = 5Gi`

   You can run your suggestion container without requesting
   the `ephemeral-storage` resource from the Kubernetes cluster.
   For instance, if you want to use
   [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
   for your Katib experiments. To remove the `ephemeral-storage` resource from
   the suggestion container, set the negative values for the

>    `ephemeral-storage` requests and limits in your Katib config:

Reviewer: Same as the comment before - PTAL
